### ISSUE TITLE
Merge Plugins and main QMake files

### ISSUE THREAD

**Issue by Timmmm** [pawelsalawa/sqlitestudio#3883]
As discussed in #3880 it would greatly simplify compilation if you didn't need to run `qmake` and `make` separately for the core and plugins projects. It should be possible to run `qmake`/`make` once and build everything.

**Comment by pawelsalawa** [comment]
Yeah... this separation comes from the very initial idea of having as much extensibility as possible, but perhaps that was a bit over exaggerated. I also think that compilation process could get some simplification here and there.
